### ISSUE TITLE
Manuscript Error: (Grammar) Missing subject in intro to Hierarchical Models

### ISSUE THREAD

**Issue by paw-lu** [BayesianModelingandComputationInPython/BookCode_Edition1#156]
# Description

This phrase in the intro to **Hierarchical Models** is missing a subject:

https://github.com/BayesianModelingandComputationInPython/BookCode_Edition1/blob/1ae02289e4b9961ad56e06de7700b1a1119d1856/markdown/chp_04.md?plain=1#L976-L979

I'm assuming it's meant to be something like hyperparameter. eg

> The partial refers to the idea that groups that do not share one fixed parameter, but share a **hyperparameter** which describes the distribution of for the parameters of the prior itself.

**Reference**
 

## Author Section
Do not close until

- [ ] Added to Errata
- [ ] Fixed in Open Access
- [ ] Fixed in latex source

---

Once again, enjoying the book here, thanks!

**Comment by canyon289** [comment]
Im looking at the printed text and it should `hyperprior distribution`. Looks like there was a rendering issue in the markdown. Thanks for pointing this out.

Note for myself. I wont add to errata since print is correct, will fix in markdown

**Comment by paw-lu** [comment]
Thanks so much for checking @canyon289!

> Looks like there was a rendering issue in the markdown.

Want to point out that if you took at the section above it links to the source code. This is not a rendering issue, but it seems the subject is missing from the source markdown itself.

`git blame` doesn't show any modifications since the initial commit on that line. So there must have been some change after the book was printed and before it open sourced here.

**Comment by canyon289** [comment]
Good detective work! You're right it wasn't in the source markdown posted here.

The TLDR basically is, we wrote the book in Latex for optimize for print. The Latex was rendered one way for printing the text. 

To get it into Jupyter book though we had to do fairly hacky series of conversions to get it to markdown and the word got lost in that second conversion process.

These are all the things no one tells you when you're writing a technical book :(
To be honest I'm very happy with the final product ended up being but also am unhappy at all the things that we could have done better. These are all learnings we're going to incorporate for the second edition.

Your issue tickets and pull requests are really helping us with that learning process. 

<img width="861" alt="image" src="https://user-images.githubusercontent.com/7213793/169706237-0d19dde5-6901-4063-ae6c-ac6d5c3e9a45.png">
